src/routes/(auth)/signup/+page.svelte

```diff
@@ -25,17 +22,13 @@
       password: Yup.string().required()
     }),
     onSubmit: async (values) => {
-      const response = await urqlClient
```

Comment:
Now this is not required anymore, due to the fact that this will be achieved server side.